src/frontend/components/UI/HowLongToBeat/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { HowLongToBeatEntry } from 'howlongtobeat'
+import React, { useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import './index.scss'
+
+// react component with 3 square with how long to beat info
+
+type Props = {
+  title: string
+}
+
+export default function HowLongToBeat({ title }: Props) {
+  const [howLongToBeatInfo, setHowLongToBeatInfo] =
+    React.useState<HowLongToBeatEntry | null>(null)
```

Comment:
let's use `useState` directly and imported in line 2, for consistency with other components and hooks

Reply:
fixed